torch_frame/data/stats.py

```diff
@@ -139,9 +159,20 @@ def compute_col_stats(
             for stat_type in StatType.stats_for_stype(stype)
         }
     else:
+        if stype == torch_frame.timestamp:
+            ser = pd.to_datetime(ser, format=time_format)
+            ser = ser.sort_values()
         stats = {
-            stat_type: stat_type.compute(ser.dropna(), sep, time_format)
+            stat_type: stat_type.compute(ser.dropna(), sep)
             for stat_type in StatType.stats_for_stype(stype)
         }
 
     return stats
+
+
+def extract_time(t: datetime) -> List[np.float32]:
```

Comment:
```suggestion
def extract_time(t: datetime) -> List[int]:
```
Why `List[np.float32]`? You can also just return a `Tensor` here?